Raven.Abstractions/Smuggler/SmugglerDatabaseApiBase.cs

```diff
@@ -533,6 +544,9 @@ public static void WriteLastEtagsToFile(OperationState result, string backupPath
                         {
                             while (await documents.MoveNextAsync())
                             {
+                                if (Options.ExportDeletions && updateLastDocDeleteEtag != null)
```

Comment:
This means that if we have 1M of documents, we will do it 1M times... why it can't be done just before ExportDeletions and only once?